website/server/models/challenge.js

```diff
@@ -209,8 +209,22 @@ schema.methods.updateTask = async function challengeUpdateTask (task) {
     updateCmd.$set[key] = syncableAttrs[key];
   }
 
```

Comment:
That's so much better, I didn't like this and then didn't think better of fixing it after I saw how the tests did it...